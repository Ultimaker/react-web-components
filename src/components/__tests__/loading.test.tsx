// Copyright (c) 2018 Ultimaker B.V.
import * as React from 'react';
import { shallow } from 'enzyme';

// component
import Loading from '../loading';

describe('The Loading component', () => {
    let props;
    let wrapper;

    beforeEach(() => {
        props = {
            label: 'loading',
        };
        wrapper = shallow(<Loading {...props} />);
    });

    it('should render', () => {
        expect(wrapper).toMatchSnapshot();
    });

});
