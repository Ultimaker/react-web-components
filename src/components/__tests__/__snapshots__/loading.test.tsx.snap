// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Loading component should render 1`] = `
<div
  className="loader"
>
  loading
  <span>
    <span
      className="loader__dot"
    >
      .
    </span>
    <span
      className="loader__dot"
    >
      .
    </span>
    <span
      className="loader__dot"
    >
      .
    </span>
  </span>
</div>
`;
